src/stratis_cli/_actions/_data.py

```diff
@@ -91,6 +91,13 @@
 <arg name="return_code" type="q" direction="out"/>
 <arg name="return_string" type="s" direction="out"/>
 </method>
+<method name="SnapshotFilesystem">
```

Comment:
Just want to make sure that this update got into the file by:
1. Firing up stratisd, creating a filesystem, and calling Introspect() on the filesystem object.
2. Overwriting whatever was previously in this file for Filesystem w/ the new value.

Otherwise, future editors may run into unnecessary conflicts.

Reply:
@mulkieran I called Instrospect on the pool (not filesystem) via:

busctl call org.storage.stratis1 /org/storage/stratis1/1 org.freedesktop.DBus.Introspectable Introspect

But the format didn't match what was in the code.  I made some format adjustments by hand.  Is there a formatting flag you passed busctl?

Reply:
I didn't use busctl directly, I used one of the GUI wrappers, w/out realizing it wasn't doing the same thing as busctl. Maybe we should have a policy. But in any case, since the order is exactly that which stratisd generates, that is good enough for now.

Reply:
Ok.  I looked at it with d-feet and it is similar - but would have also required some adjustments to match the code.

Reply:
Yeah. I used the other GUI, I forget it's name right now. It's the one @agrover recommended. Hmm.